src/main/java/teammates/ui/controller/InstructorFeedbackPreviewAsStudentAction.java

```diff
@@ -42,6 +42,8 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         data.bundle = logic.getFeedbackSessionQuestionsBundleForStudent(
                 feedbackSessionName, courseId, previewStudent.email);
         
+        data.filterSessionQuestionBundle();
```

Comment:
@unyoungwax, we can improve on the naming here to be more explicit about what this is doing.